openstack/clientconfig/requests.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/gophercloud/gophercloud/openstack"
 
 	"gopkg.in/yaml.v2"
+	"reflect"
```

Comment:
Since `reflect` is part of the standard library, it should go in between `os` and `strings`. This will fix the Travis error.

Reply:
You're right. I though a simple gofmt would suffice but apparently not 😄